x-pack/filebeat/input/salesforce/config_auth.go

```diff
@@ -33,6 +33,7 @@ type JWTBearerFlow struct {
 	Enabled *bool `config:"enabled"`
 
 	URL            string `config:"url"`
+	TokenURL       string `config:"token_url"` // If not provided, the URL will be used by go-sfdc package.
```

Comment:
```suggestion
	TokenURL       string `config:"token_url"` // If not provided, the URL field will be used as token_url by go-sfdc package.
```

Reply:
Actually, this comment is accurate. So, what is happening internally is that, if we do not provide, the URL field will be taken. But let me elaborate this description a bit to make it more clear.

PR: https://github.com/elastic/go-sfdc/pull/4/changes